jobs/worker/templates/env.sh.erb.tmpl

```diff
@@ -32,18 +32,11 @@ export CONCOURSE_GARDEN_CONFIG=/var/vcap/jobs/worker/config/garden.ini
 
 {{template "env_helpers.erb.tmpl" .}}
 
-#TODO: When the Concourse binary starts defaulting to containerd in the future the if condition should also check for
-#the case where runtime is nil (i.e. defaults to containerd without being specified)
-<%
-  if_p("runtime") do |v|
-    if v == "containerd"
--%>
+# While these only apply if the containerd runtime is used, adding logic here
+# to figure out what the default runtime is is kinda annoying, so let's just
+# set these by default. Guardian runtime will just ignore them
```

Comment:
I think at one point the worker errored if there were any env vars set for another runtime but seems like we got rid of that logic